frontends/web/src/routes/account/add/add.tsx

```diff
@@ -155,7 +159,10 @@ export const AddAccount = ({ accounts }: TAddAccountGuide) => {
           ref={inputRef}
           id="accountName"
           onInput={e => setAccountName(e.target.value)}
-          value={accountName} />
+          value={
+            accountName ||
+              (onlyOneSupportedCoin() ? supportedCoins[0].suggestedAccountName : '')
```

Comment:
Could you move this into back function and use setAccountName ?

Reply:
@thisconnect as discussed privately, I don't think `back` is the right place, but I moved the logic to `startProcess` and I think the result is much more cleaner now, wdyt?